tests/performanceplatform/client/test_admin.py

```diff
@@ -62,6 +62,25 @@ def test_create_data_set(self, mock_request):
             data=json.dumps(data_set_config))
 
     @mock.patch('requests.request')
+    def test_create_transform(self, mock_request):
+        mock_request.__name__ = 'request'
+        transform_config = {
+            'name': 'carers-allowance'
+        }
+        base_url = 'get_data_group.com'
```

Comment:
Pedantic boring comment, but its best to use [reserved domain names](http://www.iana.org/domains/reserved) like `example.com` for this sort of thing.